README.md

```diff
@@ -22,12 +22,21 @@ Let's take Unity Catalog for spin. In this guide, we are going to do the followi
   as well as provide a convenient way to explore the content of any UC server implementation.
 
 ### Prerequisites
+- Docker
```

Comment:
You do have to clone this repo still, so the 2 options has to be under the instructions of cloning the repo.

Reply:
You don't need to clone the repo if running the `docker run ...` command. It will pull the image from Dockerhub and start running straight away.

Reply:
Can you elaborate a bit more about this? what will be the full command of `docker run ... `
If we are adding a completely new way to run these steps, then we need to write down the exact steps that people can copy paste and run and leave nothing to assumption. 

And I am not as familiar with Docker as i want to be. I am guessing that this `docker run ...` command is going to download from dockerhub? who is going to publish docker hub?